lib/config/options/index.ts

```diff
@@ -618,48 +620,50 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'username',
-    description: 'Username for authentication. Currently Bitbucket only.',
+    description: 'Username for authentication.',
     stage: 'repository',
     type: 'string',
+    supportedPlatforms: ['bitbucket', 'bitbucket-server'],
```

Comment:
I've copy pasted the `supportedPlatforms` array from the `bbUseDefaultReviewers` config option in this file.

Please check if this is correct. 😉

Reply:
```suggestion
    supportedPlatforms: ['azure', 'bitbucket', 'bitbucket-server'],
```

also supported on azure